src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,151 @@
+package duke;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import duke.command.AddCommand;
+import duke.command.AddCommandType;
+import duke.command.Command;
+import duke.command.DeleteCommand;
+import duke.command.DoneCommand;
+import duke.command.ExitCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.task.TaskList;
```

Comment:
I like how neat the import statements are!

Reply:
Thank you! 😃